magento/magento2
# Fixed 24031: Layer navigation displaying whenever select Static Block only

Fiexed: Layer navigation displaying whenever select Static Block only.
### Description (*)

### Fixed Issues (if relevant)

1. #24031 

### Manual testing scenarios (*)

1. After fixings set category as display **static block only**, now only static block is displayed 
2. Check by setting category as display **static block and products**, Now block and layered navigation displayed.
3. Check by setting category as display **products only** , Now layered navigation is displayed

### Questions or comments


### Contribution checklist (*)
